netpyne/sim/load.py

```diff
@@ -306,19 +306,16 @@ def sort(popKeyValue):
                         pop.cellGids = popLoad['cellGids']
                         sim.net.pops[popLoadLabel] = pop
                     for cellLoad in cellsNode:
-                        # create new CompartCell object and add attributes, but don't create sections or associate gid yet
-                        # TO DO: assumes CompartCell -- add condition to load PointCell
-                        cell = sim.CompartCell(gid=cellLoad['gid'], tags=cellLoad['tags'], create=False, associateGid=False)
-                        try:
-                            if sim.cfg.saveCellSecs:
-                                cell.secs = Dict(cellLoad['secs'])
-                            else:
-                                createNEURONObjorig = sim.cfg.createNEURONObj
-                                sim.cfg.createNEURONObj = False  # avoid creating NEURON Objs now; just needpy struct
-                                cell.create()
-                                sim.cfg.createNEURONObj = createNEURONObjorig
-                        except:
-                            if sim.cfg.verbose: print(' Unable to load cell secs')
+                        # create new cell object and add attributes, but don't create sections or associate gid yet
+                        secs = cellLoad.get('secs', None)
+                        gid = cellLoad['gid']
+                        tags = cellLoad['tags']
+                        if secs:
+                            cell = sim.CompartCell(gid, tags, create=False, associateGid=False)
+                            cell.secs = Dict(secs)
+                        else:
+                            cell = sim.PointCell(gid, tags, create=False, associateGid=False)
```

Comment:
I use absence or presence of `secs` to distinguish between `PointCell` and `CompartCell`, is this correct? 

But in any case, we seem to be in trouble if `cfg.saveCellSecs` was switched off when model was saved. If the info about sections was anyway lost this way, is it legal to treat `CompartCell` as `PointCell` upon loaded?